dcm2bids/utils/io.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8 -*-
+
+import inspect
+import json
+import os
+import os.path as opj
+from pathlib import Path
+from collections import OrderedDict
+
+import dcm2bids
+
+
+def load_json(filename):
+    """ Load a JSON file
+
+    Args:
+        filename (str): Path of a JSON file
+
+    Return:
+        Dictionnary of the JSON file
+    """
+    with open(filename, "r") as f:
+        data = json.load(f, object_pairs_hook=OrderedDict)
+    return data
+
+
+def save_json(filename, data):
+    with open(filename, "w") as f:
+        json.dump(data, f, indent=4)
+
+
+def write_txt(filename, lines):
+    with open(filename, "a") as f:
+        for row in lines:
+            f.write("%s\n" % row)
```

Comment:
```suggestion
    with open(filename, "a+") as f:
        f.write(f"{lines}\n")
```
Otherwise there is a new line after each character